app/views/shared/_navbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "/", class: "navbar-brand" do %>
-    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= image_tag("chomp_logo.png") %>
```

Comment:
I thought we not using image any more but the ChompFood in a stylized google font?

Reply:
The text is updated on master so it's conflicting with this change. If remove this change in the conflict you can ignore making changes to this file on this branch

Reply:
![Screen Shot 2021-09-19 at 2 35 17 am](https://user-images.githubusercontent.com/84091703/133905094-54f0ca7a-1846-475f-b67d-868493c5c632.png)
Yes I believe this was before I pulled your git branch because on my end I do not have the image_tag anymore.